source/administration/changelog.md

```diff
@@ -4,6 +4,191 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v4.3.0
+Release date: 2017-10-16
+
+### Security Update
+
+- Mattermost v4.3.0 contains multiple security fixes ranging from low to high severity. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 14 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Highlights
+
+#### Data Retention Beta ([Enterprise Edition E20](https://about.mattermost.com/pricing/))
+- Automatically delete old messages and file uploads through custom data retention policies.
+
+#### Languages
+- Promoted Italian and Turkish out of beta to release-quality translations.
+
+### Improvements
+
+#### Web User Interface
+- Clicking on "More Unreads" bubble on channel sidebar now scrolls you to the next unread channel.
+- Added status indicators to direct message channel header.
+
+#### Search
+- Improved user search that supports two character full names and usernames.
+
+#### Integrations
+- Ephemeral slash command responses now support attachements without text.
+- Added CLI command to move custom slash commands between teams.
+
+#### Notifications
+- Updating the channel header with channel-wide mentions no longer triggers notifications.
+
+#### Performance
+- Improved initial load of the emoji picker, now supporting thousands of custom emojis including GIFs.
+
+#### Enterprise Edition
+- Added tables in System Console for AD/LDAP sync, Elasticsearch and Data Retention to track the success of scheduled jobs.
+- Added an idle timeout setting to automatically log out users who are inactive for a specified amount of time.
+- Elasticsearch can now be used on a shared Elasticsearch cluster.
+- Added metrics for monitoring Elasticsearch system health and usage.
```

Comment:
Did this get doc'd?

Reply:
Not sure of the details, we have this: https://github.com/mattermost/docs/commit/31cbbe2ac2c340d050f2e9034617fb8f66f2fe25#diff-5fe3eb6d5b785977f68796af61cdec13